### PR TITLE
Plane: Added manual expo input options for MANUAL, TRAINING and ACRO

### PR DESCRIPTION
**Pull request by tridge**
This makes flying in manual modes a lot easier on twitchy planes, and is easier than setting up per-mode expos on transmitters

Also adds ONESHOT_MASK for oneshot on servos in fixed wing

### CODE REVIEW

File: ArduPlane/Parameters.cpp [hunk 1]
```diff
@@ -1225,6 +1225,37 @@ const AP_Param::GroupInfo ParametersG2::var_info[] = {
     AP_SUBGROUPINFO(guidedHeading, "GUIDED_", 28, ParametersG2, AC_PID),
 #endif // OFFBOARD_GUIDED == ENABLED
 
+    // @Param: MAN_EXPO_ROLL
+    // @DisplayName: Manual control expo for roll
+    // @Description: Percentage exponential for roll input in MANUAL, ACRO and TRAINING modes
+    // @Range: 0 100
```
Comment (Gone4Dirt) [inline ArduPlane/Parameters.cpp:1231]: Not a big issue, but I think these should be in the range 0 to 1 for better alignment with expo params in copter.  Just to keep them consistant between vehicles.

**Review by Gone4Dirt — COMMENTED** [review]
LGTM.  The expo is a nice feature.
Not a big issue but IMHO the expo params should be in the range 0-1 to match Copter expo params

File: ArduPlane/Parameters.cpp (hunk 1 above)
Comment (Hwurzburg) [reply ArduPlane/Parameters.cpp:1231]: And in  transmitters

**Comment by tridge** [comment]
>  Not a big issue but IMHO the expo params should be in the range 0-1 to match Copter expo params

it is supposed to match the expo used on a transmitter. That is always integer 0 to 100

**Comment by IamPete1** [comment]
I have a PR in the works for rover adding expo, there I used a range of -0.5 to 1 to match copter's `ACRO_RP_EXPO` and `ACRO_Y_EXPO`. Planes existing `Q_THROTTLE_EXPO` is also in the range 0 to 1.

 I think we should keep constant across AP with the 0 to 1 range.

File: ArduPlane/system.cpp [hunk 2]
```diff
@@ -120,7 +120,11 @@ void Plane::init_ardupilot()
     // don't initialise aux rc output until after quadplane is setup as
     // that can change initial values of channels
     init_rc_out_aux();
-    
+
+    if (g2.oneshot_mask != 0) {
```
Comment (IamPete1) [inline ArduPlane/system.cpp:124]:
```suggestion
    if (g2.oneshot_mask > 0) {
```

Comment (peterbarker) [reply ArduPlane/system.cpp:124]: .... what's this doing in this PR, anyway?

**Comment by peterbarker** [comment]
image

**Comment by tridge** [comment]
I'm happy with the RF9 testing of the small change to use deadzone in acro

**Pull request merged**